project/src/main/world/Overworld.tscn

```diff
@@ -486,41 +493,70 @@ shadow_scale = 4.0
 [node name="Sprite" parent="World/Obstacles/ButtercupRestaurant" index="0"]
 material = SubResource( 4 )
 
-[node name="ButtercupStool1" parent="World/Obstacles" instance=ExtResource( 43 )]
+[node name="ButtercupStool1" parent="World/Obstacles" instance=ExtResource( 52 )]
 position = Vector2( 2493, 928 )
-shadow_scale = 0.6
-occupied_texture = ExtResource( 47 )
-unoccupied_texture = ExtResource( 44 )
-
-[node name="ButtercupStool4" parent="World/Obstacles" instance=ExtResource( 43 )]
-position = Vector2( 3050.06, 765.483 )
-shadow_scale = 0.6
-occupied_texture = ExtResource( 47 )
-unoccupied_texture = ExtResource( 44 )
-
-[node name="ButtercupStool2" parent="World/Obstacles" instance=ExtResource( 43 )]
-position = Vector2( 2714.92, 927.829 )
-shadow_scale = 0.6
-occupied_texture = ExtResource( 46 )
-unoccupied_texture = ExtResource( 45 )
-
-[node name="ButtercupStool3" parent="World/Obstacles" instance=ExtResource( 43 )]
-position = Vector2( 2837.62, 771.146 )
-shadow_scale = 0.6
-occupied_texture = ExtResource( 46 )
-unoccupied_texture = ExtResource( 45 )
-
-[node name="ButtercupTable1" parent="World/Obstacles" instance=ExtResource( 13 )]
+overworld_world_path = NodePath("../..")
+target_properties = {
+"occupied_texture": ExtResource( 47 ),
+"unoccupied_texture": ExtResource( 44 )
+}
+spawn_if = "not level_finished marsh/goodbye_everyone"
+
+[node name="ButtercupStool4" parent="World/Obstacles" instance=ExtResource( 52 )]
+position = Vector2( 3050, 765 )
+overworld_world_path = NodePath("../..")
+target_properties = {
+"occupied_texture": ExtResource( 47 ),
+"unoccupied_texture": ExtResource( 44 )
+}
+spawn_if = "not level_finished marsh/goodbye_everyone"
+
+[node name="ButtercupStool2" parent="World/Obstacles" instance=ExtResource( 54 )]
+position = Vector2( 2715, 928 )
+overworld_world_path = NodePath("../..")
+target_properties = {
+"occupied_texture": ExtResource( 46 ),
+"unoccupied_texture": ExtResource( 45 )
+}
+spawn_if = "not level_finished marsh/goodbye_everyone"
+
+[node name="ButtercupStool3" parent="World/Obstacles" instance=ExtResource( 54 )]
+position = Vector2( 2838, 771 )
+overworld_world_path = NodePath("../..")
+target_properties = {
+"occupied_texture": ExtResource( 46 ),
+"unoccupied_texture": ExtResource( 45 )
+}
+spawn_if = "not level_finished marsh/goodbye_everyone"
+
+[node name="ButtercupTable1" parent="World/Obstacles" instance=ExtResource( 43 )]
 position = Vector2( 2603.08, 980.722 )
-shadow_scale = 1.2
```

Comment:
The ButtercupTable.tscn should have its shadow_scale set to 1.2.

Reply:
![image](https://user-images.githubusercontent.com/5902413/126684463-109eb3f3-bd58-4f01-b646-daea4fc4b3a4.png)
![image](https://user-images.githubusercontent.com/5902413/126684467-103eb6c7-f42c-4df5-846e-47343da1313f.png)


Looking at both versions, I like the original value of `0.9` better -- i'm leaving it.